core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -385,6 +385,34 @@
       + "  ]\n"
       + "}";
 
+  public static final String XX4 = "{\n"
+      + "  \"rels\": [\n"
+      + "    {\n"
+      + "      \"id\": \"0\",\n"
```

Comment:
Please use a more readable name.

Reply:
The name follows the style above, I'll update it